File: server/Models/Keep.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace keeper.Models;

public class Keep : RepoItem<int>
{
    public string CreatorId { get; set; }
    [MaxLength(255)] public string Name { get; set; }
    [MaxLength(1000)] public string Description { get; set; }
    [Url] public string Img { get; set; }
    public int Views { get; set; }
    public int Kept { get; set; }
    public Account Creator { get; set; }
}

File: client/src/models/Keep.js

export class Keep{
constructor(data){
    this.id = data.id
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
    this.description = data.description
    this.name = data.name
    this.img = data.img
    this.kept = data.kept
    this.views = data.views
    this.creatorId = data.creatorId
    this.creator = data.creator
}
}